typings/index.d.ts

```diff
@@ -2725,7 +2725,54 @@ export type BitFieldResolvable<T extends string, N extends number | bigint> =
 
 export type BufferResolvable = Buffer | string;
 
-export type CacheFactory = <T>(manager: { name: string }, holds: { name: string }) => Collection<Snowflake, T>;
+export type CachedManagerTypes =
```

Comment:
This could be `keyof CacheFactoryArgs`, but it does not seem to be used anywhere (anymore?)?

Reply:
This what? The generic type `T`?

Reply:
Ohh, got it. You're right!